snippets/resolver/named.conf

```
@@ -1,28 +1,48 @@
 options {
-  directory "/var/cache/bind";
-  listen-on { any; };
-  listen-on-v6 { any; };
-  pid-file "/run/named/named.pid";
+directory "/var/cache/bind";
+allow-recursion {
+192.168.0.0/24;
+127.0.0.1/32;
 };
-
-zone "2tm1-admin-sys2.ephec.austreelis.net" IN {
-  type master;
-  file "/etc/bind/2tm1-admin-sys2.ephec.austreelis.net.zone";
+// Configure the IPs to listen on here.
+listen-on port 53 {any;};
+listen-on-v6 { none; };
+pid-file "/var/run/named/named.pid";
+allow-transfer { none; };
+};
+zone "." IN {
+type hint;
+file "/usr/share/dns/root.hints";
+};
+zone "localhost" IN {
+type master;
+file "db.local";
+allow-update { none; };
+notify no;
+};
+zone "127.in-addr.arpa" IN {
+type master;
+file "db.127";
+allow-update { none; };
+notify no;
```

Fix indentation:

```suggestion
zone "127.in-addr.arpa" IN {
  type master;
  file "db.127";
  allow-update { none; };
  notify no;
```

Same as the previous 2, that db file isn't included in the docker image. Also, I'm not sure we need such a reverse mapping for a resolver ?

---

Same as the previous